cpp/include/cudf/strings/string_view.cuh

```diff
@@ -298,8 +296,9 @@ class string_view
   __device__ size_type rsplit( const char* delimiter, size_type count, string_view* strs ) const;
 
 private:
-    const char* _data{};   ///< Pointer to device memory contain char array for this string
-    size_type _bytes{};    ///< Number of bytes in _data for this string
+    const char* _data{};           ///< Pointer to device memory contain char array for this string
+    size_type _bytes{};            ///< Number of bytes in _data for this string
+    mutable size_type _nchars{};   ///< Number of characters in this string (computed)
```

Comment:
Why not just call this `_length`?  The `n` prefix is inconsistent with `_bytes`.